bin/wmagent-resource-control

```diff
@@ -109,10 +109,11 @@ def getCMSSiteInfo(pattern):
     for seName in siteDB.cmsNametoList(pattern, "SE", "result.json"):
         cmsNames = siteDB.seToCMSName(seName)
```

Comment:
actually we can call siteDB.PSNtoPNN call directly to get this mapping. (Although the list is not complete there but I think site db need be updated for that)
under lining call is 
https://cmsweb.cern.ch/sitedb/data/prod/data-processing